### ISSUE TITLE
Frontend in 2.22.x not working 

### ISSUE THREAD

**Issue by Vrtak-CZ** [SolidStateGroup/bullet-train-api#1026]
Looks like there is some problem with forwarding urls to index in SPA I'm getting 404 when accessing `https://our-flagsmith.domain/projects/`

```
flagsmith-api-7ffb9b4b4f-dxtzc django.request WARNING  Not Found: /projects/
```

**Comment by Vrtak-CZ** [comment]
We have this issue on 2.22.0 and on 2.22.1

**Comment by dabeeeenster** [comment]
Are you able to provide steps to reproduce?

**Comment by Vrtak-CZ** [comment]
We have self-hosted flagsmith deployed with `separateApiAndFrontend: false` and after upgrade to 2.22.0 or 2.22.1 I open up frontend on `https://our-flagsmith.domain` it works but it makes "redirect" to `https://our-flagsmith.domain/projects/` and when I trigger refresh in browser I got 404.

**Comment by dabeeeenster** [comment]
This is part of a Helm/Kubernetes deployment? Which version of the helm chart?

**Comment by Vrtak-CZ** [comment]
Helm chart 0.7.2

**Comment by Vrtak-CZ** [comment]
but as I said with image tag 2.21.2 it works ok

**Comment by dabeeeenster** [comment]
OK thanks thats helpful - think I know what this is - will take a look

**Comment by Vrtak-CZ** [comment]
Thanks. Also thanks for quick response.

**Comment by dabeeeenster** [comment]
Fairly sure this is related to https://github.com/Flagsmith/flagsmith/pull/984

**Comment by matthewelwell** [comment]
Hi @Vrtak-CZ, thanks for the report here. This issue should be resolved in 2.22.2 which I have just released.

**Comment by Vrtak-CZ** [comment]
Hi @matthewelwell, looks like 2.22.2. Thank you.
